post: cfp launch announcement (#611)
pyladies/pyladies: www/_posts/2023/2023-07-28-cfp-launch-announcement.md

---
layout: post.html
title: "Announcement: CFP Launch"
tags: [PyLadies, Conference, Global]
author: Denny Perez
author_link: https://twitter.com/dennyperez18
---

**📢 Call for Paper Launch: PyLadiesCon - Share Your Insights with Us!**

Dear PyLadies community members,

As we venture further into the journey of PyLadiesCon, we are thrilled to announce the official launch of our Call for Papers.

**🎉 What is PyLadiesCon?**

PyLadiesCon is an extraordinary event designed to promote diversity, learning, and empowerment within the Python community. It celebrates women in Python, aiming to create a welcoming and inclusive environment for all attendees.

**🌟 Share Your Knowledge: Call for Papers**

We invite speakers from all backgrounds and experiences to share their insights, expertise, and stories with our vibrant community. Whether you're a seasoned Python developer, an industry expert, or a first-time speaker, we encourage you to submit your proposals!

As we are a global community, we welcome talks in the following languages: **English, Español, Português, Français, and 中文**. 

**🔗 Submit Your Proposal:**

> [Please click here to submit your proposal.](https://pretalx.com/orga/event/pyladiescon-2023/live)

Reviewer: ```suggestion
[Please click here to submit your proposal.](https://pretalx.com/pyladiescon-2023/cfp)
```

> As we are a global community, you are welcome to present in any one of the following languages: \*\*English, Spanish, Portuguese, French, and Chinese.\*\*. However, due to the limitation of our proposal review team, we can only review talk proposals in the following languages: “English, Spanish, Portuguese, and French”. Please indicate in your proposal which language you will present in, if it is different than the language of the proposal.

Reviewer: ```suggestion
As we are a global community, we welcome talks in the following languages: **English, Español, Português, Français, and 中文**. 
```

> Please click here to submit your proposal. \[https://pretalx.com/orga/event/pyladiescon-2023/live\]

Reviewer 2: with that fixed I believe this can be merged no problem ;)

The topics can range from technical talks to best practices, case studies, and more. Every story and perspective is valuable, and your contribution will enrich the conference experience for everyone.

**🗓️ Important Dates:**

> Call for Papers Launch: July 27th

Reviewer: ```suggestion
Call for Papers Launch: August 4th
```
